### PR TITLE
Add fpetkovski to triage list

### PR DESCRIPTION
**Pull request by fpetkovski**
* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes

Add myself to triage.

## Verification

Lost of unit and e2e tests. Benchmarks will follow soon.

### CODE REVIEW

**Review by kakkoyun — APPROVED** [review]
You need to sign the commits, you know the drill ;) Otherwise LGTM :) Welcome!

**Comment by fpetkovski** [comment]
Looks like it's time to automate the signoff :)

File: MAINTAINERS.md [hunk 1]
```diff
@@ -28,15 +28,16 @@ We also have some nice souls that help triaging issues and PRs. See [here](https
...
+| Name            | Slack              | GitHub                                       | Company |
+|-----------------|--------------------|----------------------------------------------|---------|
+| Adrien Fillon   | `@Adrien F`        | [@adrien-f](https://github.com/adrien-f)     |         |
+| Ian Billett     | `@billett`         | [@bill3tt](https://github.com/bill3tt)       | Red Hat |
+| Martin Chodur   | `@FUSAKLA`         | [@fusakla](https://github.com/fusakla)       |         |
+| Michael Dai     | `@jojohappy`       | [@jojohappy](https://github.com/jojohappy)   |         |
+| Xiang Dai       | `@daixiang0`       | [@daixiang0](https://github.com/daixiang0)   |         |
+| Jimmie Han      | `@hanjm`           | [@hanjm](https://github.com/hanjm)           | Tencent |
+| Matej Gera      | `@Matej Gera`      | [@matej-g](https://github.com/matej-g)       | Red Hat |
+| Filip Petkovski | `@Filip Petkovski` | [@fpetkovski](https://github.com/fpetkovski) |         |
```
Comment (yeya24) [inline MAINTAINERS.md:40]: Is it okay to add your company name? Not sure why many team members don't have a company name here.

Comment (fpetkovski) [reply MAINTAINERS.md:40]: I would prefer to omit it for now due to temporary internal policy that we have in place. I'll add it in several months once the policy has expired :)

**Review by matej-g — APPROVED** [review]
Welcome on board! :ship:

**Pull request merged**